README.rst

```diff
@@ -72,3 +67,219 @@ response, e.g.
+        # the URLs to the files. Set https as default.
+        # https://github.com/jschneier/django-storages/blob/10d1929de5e0318dbd63d715db4bebc9a42257b5/storages/backends/s3boto3.py#L217
+        AWS_S3_URL_PROTOCOL = os.environ.get("AWS_S3_URL_PROTOCOL", "https:")
+
+You can set the following environment variables to configure your bucket if you
+used the above snippet.
+
+* ``AWS_STORAGE_BUCKET_NAME`` - set to ``media.llamasavers.com``.
+* ``AWS_S3_CUSTOM_DOMAIN`` - set to ``media.llamasavers.com``.
+* ``AWS_S3_REGION_NAME`` - set to your AWS region name, e.g. ``eu-west-2``.
+
+You can use one of the methods to provide `boto3 with credentials`__. E.g. to
+use environment variables, you need to set the following variables:
+
+* ``AWS_ACCESS_KEY_ID``
+* ``AWS_SECRET_ACCESS_KEY``
+
+.. _Boto3Credentials: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
+
+__ Boto3Credentials_
+
+Now the storage should be configured and working, e.g. editors should be able to
+upload images and documents in Wagtail admin.
+
+Set up ``wagtail-storages``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Install ``wagtail-storages`` itself.
+
+.. code:: sh
+
+   pip install wagtail-storages
+
+
+Add ``wagtail_storages`` to your ``INSTALLED_APPS`` in your settings file.
+
+.. code:: python
+
+   # settings.py
+
+   INSTALLED_APPS = [
+       # ... Other apps
+       "wagtail_storages",
+       # ... Other apps
+   ]
+
+With having that set up that, ACLs should be updated if documents are moved to
+private collections.
+
+Set up front-end cache invalidation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If edge cache is set up on the custom domain, ``media.llamasavers.com``, the CDN
+purge should be set up to avoid having outdated or private documents available
+to users via the CDN endpoint. For example, for Cloudflare you want to use a
+configuration similar to the one below:
+
+.. code:: python
+
+   # settings.py
+   import os
+
+
+   if "S3_CACHE_CLOUDFLARE_TOKEN" in os.environ:
+        WAGTAIL_STORAGES_DOCUMENTS_FRONTENDCACHE = {
+            "default": {
+                "BACKEND": "wagtail.contrib.frontend_cache.backends.CloudflareBackend",
+                "EMAIL": os.environ["S3_CACHE_CLOUDFLARE_EMAIL"],
+                "TOKEN": os.environ["S3_CACHE_CLOUDFLARE_TOKEN"],
+                "ZONEID": os.environ["S3_CACHE_CLOUDFLARE_ZONEID"],
+            },
+        }
+
+Then setup the following environment variables:
+
+* ``S3_CACHE_CLOUDFLARE_EMAIL``
+* ``S3_CACHE_CLOUDFLARE_TOKEN``
+* ``S3_CACHE_CLOUDFLARE_ZONEID``
+
+By having this set up, the documents will be purged from cache when they are
+modified or their privacy settings have changed.
+
+The setting follows configuration format of Wagtail's module -
+``wagtail.contrib.frontend_cache``. See the details `here`__. The only
+difference is the setting name.
+
+.. _WagtailFrontEndCache: https://docs.wagtail.io/en/stable/reference/contrib/frontendcache.html
+
+__ WagtailFrontEndCache_
```

Comment:
You could put a final message at the bottom, to the effect of 'All done!'

Obvs not a requirement :)